OneCommunityGlobal/HighestGoodNetworkApp
# Shantanu unit test cases NewModal File

# Description
Unit test for common New Modal component


## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. run on terminal: "npm test Image.test.jsx"
4. verify that all tests pass

## Screenshots or videos of changes:

![1](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/85691749/958e5169-0226-41a6-b1b5-9b3b14a76d46)

## Note:
Include the information the reviewers need to know.
